docs/import/add_disaster.js

```diff
@@ -370,112 +376,76 @@ function populateStateAssetPickers() {
-  for (const state of states) {
-    if (stateAssets.get(state)) {
-      assets = assets.concat(stateAssets.get(state));
-    }
-  }
+function initializeScoreSelectors(states) {
+  const headerRow = $('#score-asset-header-row');
+  const tableBody = $('#asset-selection-table-body');
+  tableBody.empty();
+  headerRow.empty();
 
-  if (assets.length) {
-    $('#score-asset-selection').show();
-  } else {
-    $('#score-asset-selection').hide();
+  // Initialize headers.
+  headerRow.append(createTd().html('Score Assets'));
+  for (const state of states) {
+    headerRow.append(createTd().html(state + ' Assets'));
   }
+  headerRow.append(createTd().html('Column Names'));
 
-  for (let i = 0; i < assets.length; i++) {
-    const asset = assets[i];
-    const name = asset.replace(/^.*(\\|\/|\:)/, '');
+  // For each asset type, add select for all assets for each state.
+  for (let scoreAssetType in scoreAssetTypes) {
     const row = $(document.createElement('tr')).attr({
-      id: name + '-row',
+      id: scoreAssetType + '-row',
     });
-    row.change(() => handleScoreAssetSelection(asset, name, row));
-    row.append($(document.createElement('td'))
-                   .append($(document.createElement('div'))
-                               .attr({
-                                 id: name + '-title',
-                               })
-                               .html(asset)));
-    row.append(createRadioButtonCell(name, 'poverty'));
-    row.append(createRadioButtonCell(name, 'svi'));
-    row.append(createRadioButtonCell(name, 'income'));
-    row.append(createRadioButtonCell(name, 'none', true));
-    row.append($(document.createElement('td'))
-                   .append($(document.createElement('div')).attr({
-                     id: name + '-columns',
-                   })));
-    $('#asset-selection-table-body').append(row);
-  }
-}
-
-/**
- * Handles the user selecting a type for one of the possible score assets.
- * @param {String} asset The asset path
- * @param {String} name The name of the row
- */
-function handleScoreAssetSelection(asset, name) {
-  const selectedType = $('input[name=\'' + name + '-select\']:checked').val();
-  const columnCell = $('#' + name + '-columns');
-  columnCell.empty();
-  if (selectedType != 'none') {
-    columnCell.append($(document.createElement('input')).attr({
-      id: name + '-column',
-      type: 'text',
-    }));
-    if (selectedType == 'poverty') {
+    row.append(createTd().append(
+        $(document.createElement('div')).html(scoreAssetType)));
+    for (const state of states) {
+      if (stateAssets.get(state)) {
+        row.append(createTd().append(createAssetDropdown(
+            stateAssets.get(state), scoreAssetType, state)));
+      }
+    }
+    // TODO: Make column names selects instead.
+    const columnCell = createTd();
+    for (let i = 0; i < scoreAssetTypes[scoreAssetType].length; i++) {
       columnCell.append($(document.createElement('input')).attr({
```

Comment:
can we create labels for these inputs? Or like text that comes up if you hover? Helpful for the poverty asset I would think.